### PR TITLE
stable-2.3 | kata-monitor: increase delay before syncing with the container manager

### PR DESCRIPTION
**Pull request by fgiudici**
This is meant to address #3550 on the stable branch.

### Proposal
We are stable, so we probably don't want to backport #3553 here.
Since we use the sync with the container manager basically to just remove pods from the cache, we can wait some more time before syncing (and so reduce the chance to miss a kata pod just because it was not ready yet) without missing reporting any metric. The sync will just clean-up the sandbox cache from stale entries.

So, let's just raise the waiting time before starting the sync timer to an higher value: this will not fix the issue (if a pod has not completely started when the sync timer fires, we will miss its metrics till a new kata pod starts / is removed) but will make the issue to be much more unlikely to happen.

### CODE REVIEW

**Review by c3d — APPROVED** [review]
Looks good to me.

Wonder if we could make that a configurable option in a later iteration?

**Comment by fidencio** [comment]
A head's up that may affect your PR.

In the Architecture Committee meeting from January 25th, 2022, the Architecture Committee has agreed on using the "Dismiss stale pull request approvals when new commits are pushed" configuration from GitHub.  It basically means that if your PR has been rebased or updated, the approvals given will be erased.

In order to minimize traumas due to the new approach, please, consider adding a note on the changes done before the rebase / force-push, and also pinging the reviewers for a subsequent round of reviews.

Thanks for your understanding!

Related issue: https://github.com/kata-containers/community/issues/249

**Comment by fgiudici** [comment]
Thanks for reviewing!

> Wonder if we could make that a configurable option in a later iteration?

Yeah, I had that thought too.
I think that the proper fix is to stick with the sbs fs to populate the sandbox cache. In this way we could even completely drop the sync with the container manager: looked a bit too intrusive for a stable branch ;-) (in #3553 we just use the container manager to get the kubernetes metadata to attach to the metrics).
Here my idea is to just do the bare minimum, i.e., put a delay high enough to be confident that the pod will be running once we check the container manager (*if* no issues on startup happen).
I would just leave this static and move to the #3553 solution (adding a new parameter here shouldn't be that hard anyway).

**Review by fidencio — APPROVED** [review]
lgtm as a workaround and I do agree it's not worth pulling the full fix down to the stable branch.
Thanks, @fgiudici!

**Pull request merged**
